src/components/ErrorMessage.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FunctionComponent, ReactNode } from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+
+import "lbh-frontend/lbh/components/lbh-error-message/_error-message.scss";
+
+export type ErrorMessageProps = {
+  id?: string;
+  className?: string;
+  visuallyHiddenText: string;
+  children?: ReactNode;
+};
+
+export const ErrorMessage: FunctionComponent<ErrorMessageProps> & {
+  defaultProps: Partial<ErrorMessageProps>;
```

Comment:
This shouldn't be necessary.

Reply:
I resolved this by reverting back to having it as an optional prop. It think that's clearer about the intent than having to do this override.